generate_export_options.rb

```diff
@@ -98,6 +98,7 @@ def export_method(mobileprovision_content)
 
 export_options = {}
 export_options[:method] = method unless method.nil?
+export_options[:uploadBitcode] = false
 
```

Comment:
No problem at all, I added this as a note for ourselves too, so that we don't accidentally merge a change which can break the step for certain use cases. 

PR: either delete and recreate on a new branch, or rebate and drop the last commit, or create a new branch and cherry pick the right commits. Cherry pick would work to separate it into two PRs (two branches) as well 😉